<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>udg::QMPR2DExtensionBase</class>
 <widget class="QWidget" name="udg::QMPR2DExtensionBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1398</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form1</string>
  </property>
  <layout class="QGridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QToolButton" name="m_slicingToolButton">
           <property name="text">
            <string>Slicing</string>
           </property>
           <property name="icon">
            <iconset resource="../../../main/main.qrc">
             <normaloff>:/images/slicing.png</normaloff>:/images/slicing.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="m_zoomToolButton">
           <property name="text">
            <string>Zoom</string>
           </property>
           <property name="icon">
            <iconset resource="../../../main/main.qrc">
             <normaloff>:/images/zoom.png</normaloff>:/images/zoom.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="m_polylineROIToolButton">
           <property name="text">
            <string>ROI</string>
           </property>
           <property name="icon">
            <iconset resource="../../../main/main.qrc">
             <normaloff>:/images/polyline.png</normaloff>:/images/polyline.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="m_distanceToolButton">
           <property name="text">
            <string>Distance</string>
           </property>
           <property name="icon">
            <iconset resource="../../../main/main.qrc">
             <normaloff>:/images/distance.png</normaloff>:/images/distance.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="m_eraserToolButton">
           <property name="text">
            <string>Rubber</string>
           </property>
           <property name="icon">
            <iconset resource="../../../main/main.qrc">
             <normaloff>:/images/eraser2.png</normaloff>:/images/eraser2.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="m_screenShotToolButton">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../../../main/main.qrc">
             <normaloff>:/images/photo.png</normaloff>:/images/photo.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="m_voxelInformationToolButton">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../../../main/main.qrc">
             <normaloff>:/images/voxelInformation.png</normaloff>:/images/voxelInformation.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="m_imagePlaneProjectionToolButton">
           <property name="text">
            <string>Image plane projection</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../../../main/images/imagePlaneProjection.png</normaloff>../../../main/images/imagePlaneProjection.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Minimum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QSlider" name="m_numberOfImagesSlider">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="m_numberOfImagesLabel">
             <property name="text">
              <string>1</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="m_thickReconstructionLabel">
           <property name="text">
            <string>Number of Images</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Minimum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="udg::QThickSlabWidget" name="m_thickSlabWidget" native="true"/>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Minimum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>1</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QToolButton" name="m_horizontalLayoutToolButton">
         <property name="text">
          <string>...</string>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="m_phasesAlertLabel">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;&quot;&gt;

&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600; color:#dc143c;&quot;&gt;*The current Series has multiple phases. &lt;/span&gt;&lt;/p&gt;

&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-weight:600; color:#dc143c;&quot;&gt;Currently the MPR 2D doesn't support &lt;/p&gt;

&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-weight:600; color:#dc143c;&quot;&gt;Series with multiple phases so it won't work propperly.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>261</width>
         <height>26</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>WW/WL:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="udg::QWindowLevelComboBox" name="m_windowLevelComboBox"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QSplitter" name="m_horizontalSplitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QSplitter" name="m_verticalSplitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QWidget" name="layout4">
       <layout class="QVBoxLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QSlider" name="m_axialSlider">
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="maximum">
             <number>0</number>
            </property>
            <property name="tracking">
             <bool>true</bool>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="tickPosition">
             <enum>QSlider::TicksBelow</enum>
            </property>
            <property name="tickInterval">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="m_axialSpinBox"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="udg::Q2DViewer" name="m_axial2DView" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <widget class="udg::Q2DViewer" name="m_sagital2DView" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="udg::Q2DViewer" name="m_coronal2DView" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>udg::Q2DViewer</class>
   <extends>QWidget</extends>
   <header>q2dviewer.h</header>
  </customwidget>
  <customwidget>
   <class>udg::QWindowLevelComboBox</class>
   <extends>QComboBox</extends>
   <header>qwindowlevelcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>udg::QThickSlabWidget</class>
   <extends>QWidget</extends>
   <header>qthickslabwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="mpr2d.qrc"/>
  <include location="../../../main/main.qrc"/>
  <include location="../../../main/main.qrc"/>
 </resources>
 <connections/>
</ui>
